model/fn-execution/src/main/proto/beam_interactive_api.proto

```diff
@@ -0,0 +1,106 @@
+/*
```

Comment:
Moved to model/interactive. Also, this file can work for Batch and Streaming. Technically, it will only be used for Batch pipelines because the interactive runner will be running streaming pipelines as Batch during re-run.